inc/bluetooth/ExternalEvents.h

```diff
@@ -33,6 +33,9 @@ DEALINGS IN THE SOFTWARE.
 #define MICROBIT_ID_BLE             1000
 #define MICROBIT_ID_BLE_UART        1001
 
+#define MICROBIT_BLE_CONNECTED      1
```

Comment:
Hi @bluetooth-mdw,

Great PR, just a few minor things

We have an unofficial connection in terms of event codes, for example:
`MICROBIT_COMPASS_EVT_CAL_END`

If both `MICROBIT_BLE_CONNECTED` and `MICROBIT_BLE_DISCONNECTED ` could be renamed that would be great!